### DISCUSSION TITLE
Is there a way to show profile of all contributors to a doc page in docusaurus?

### DISCUSSION THREAD

**Discussion by pranay01** [facebook/docusaurus#6194] [Q&A]
Is there a way to show profile of all contributors to a doc page in docusaurus?

Something like this at the bottom of each page
![Screenshot 2021-12-25 at 8 21 06 PM](https://user-images.githubusercontent.com/504541/147387641-8dfbe077-5876-4653-9fd2-998f2f29010b.png)

reference link - https://posthog.com/docs/self-host/deploy/aws

**Comment by Josh-Cena** [comment]
Not currently possible, but it's certainly on our feature request backlog! Could you send a feature request issue?

**Reply by pranay01** [reply]
Ok, should I create in canny or just an issue in github?

**Reply by Josh-Cena** [reply]
Just a GitHub issue
